tests/modules/mesoscope_splitting/test_checks.py

```diff
@@ -83,3 +85,20 @@ def mock_tiff_header_reader(arg):
               for tiff, roi_index in zip(mock_tiff_list, roi_indices)]
     with context:
         checks.splitting_consistency_check(mylist)
+
+
+class MockMesoscopeTiff():
+    def __init__(self, path):
+        return
+
+    @property
+    def plane_scans(self):
+        return np.array([-25, 15, 80, 90, 15, 120, 130, 180])
+
+
+def test_check_for_repeated_planes(monkeypatch, tmpdir):
+    placeholder = Path(tmpdir / "tmp.tiff")
+    monkeypatch.setattr(checks, "MesoscopeTiff", MockMesoscopeTiff)
```

Comment:
Would prefer `check_for_repeated_planes` just accept a `MesoscopeTiff` instance. That way you don't have to go out of your way to do convoluted monkey patching and just directly pass in your `MockMesoscopeTiff` instance.

The `MesoscopeTiff` class has a `self._source` [attribute](https://github.com/AllenInstitute/ophys_etl_pipelines/blob/5b7dc222b4f192451fa9ef13ccbfba5023425702/src/ophys_etl/modules/mesoscope_splitting/tiff.py#L127) which is equivalent to `timeseries_tiff_path` and can be used instead in the raised `ValueError` message

Reply:
done